src/resources/i18n/en.json

```diff
@@ -35,7 +35,7 @@
       },
       "weeks": {
         "current": "This week",
-        "previous": "Last Weeks",
+        "previous": "Last Week",
         "last2": "1-2 weeks",
         "last3": "1-3 weeks"
```

Comment:
These should also be translated

Reply:
@mwithi , Intesys uses transifex for translations; on the frontend side, all we need to to is to add english keys.

Reply:
I it's OK for you, let me know, so I mark this conversation as resolved.

Reply:
ops, sorry, I thought it was the code (I oversight it😆)
It's ok then! Thanks!